### ISSUE TITLE
New database model 

### ISSUE THREAD

**Issue by DonatJR** [ThirtySomething/Weatherstation#2]
@ThirtySomething In #1 ([this comment](https://github.com/ThirtySomething/Weatherstation/pull/1#discussion_r178947296)) you wanted to create a new ERM model, this issue is just for tracking the status on this. I'd be glad to have a look at it once it is finished. (There were some database related changes on the master branch - maybe this issue is already obsolete?!)

**Comment by ThirtySomething** [comment]
@DonatJR  Waiting for your review of commit 2d00613 - a rough improvement of the SQLite class.

**Comment by DonatJR** [comment]
Reviewing single commits is a little unusal. If these changes were developed on a seperate branch you could have opened a PR and added me as a reviewer :)

Is there some documentation for the ERM / database design or just the code? I can of course take a look at it but I fear potential flaws in the design might be harder to notice when looking at code alone.

**Comment by ThirtySomething** [comment]
You're right. I've to dig into the usage of the new GitHub client to understand how this works. Anyway, I've added an graphical overview to the readme.md.

**Comment by DonatJR** [comment]
The code is already merged so creating a PR is not possible anymore. If you want we can reopen this issue and I will have a look at the graphical overview?

**Comment by ThirtySomething** [comment]
As you recommended, I've reopened the issue.

**Comment by ThirtySomething** [comment]
By the way, I cannot add you as reviewer for commits done by me as long as you aren't added as a collaborator. Can you please accept the invitation?

Edit:
There is a new pull request and I like to add you as reviewer.

**Comment by DonatJR** [comment]
I accepted your invitation :)

**Comment by ThirtySomething** [comment]
I see - for the open pull request I was able to request a review. But for this issue it seems not to be possible. Do you have any hint?

**Comment by DonatJR** [comment]
There is no concept of a reviewer for an issue. Normalle an issue is an as of yet unsolved problem and gets assignees who are responsible for solving the issue. After they solved it they create a PR which gets reviewed by someone else. I can assign myself to this issue (but you should be able to change this as well)

**Comment by ThirtySomething** [comment]
Closed with the new database plugin supporting SQLite and MariaDB. See cc0ebad94dd0b17580d03c9433ac8b70554781f2 for more details.
